harvey/setrun.py

```diff
@@ -314,50 +312,116 @@ def setrun(claw_pkg='geoclaw'):
-                                     rundata.clawdata.t0,
-                                     rundata.clawdata.tfinal])
+    regions.append([1, 3, clawdata.t0, clawdata.tfinal, clawdata.lower[0], clawdata.upper[0], clawdata.lower[1], clawdata.upper[1]])
+    regions.append([1, 5, clawdata.t0, clawdata.tfinal, -99, -91, 26.5, 32]) # More specific domain f/ LaTex domain, minlevel = 1, maxlevel > 3, e.g. Can custommize based on time domain
 
-    #Port O'Connor
-    rundata.gaugedata.gauges.append([2, -96.38, 28.45,
-                                     rundata.clawdata.t0,
-                                     rundata.clawdata.tfinal])
 
-    #Aransas Wildlife Refuge
-    rundata.gaugedata.gauges.append([3, -96.75, 28.22,
-                                     rundata.clawdata.t0,
-                                     rundata.clawdata.tfinal])
 
-    #Port Aransas
-    rundata.gaugedata.gauges.append([4, -97.05, 27.84,
-                                     rundata.clawdata.t0,
-                                     rundata.clawdata.tfinal])
+	# Ruled rectangles -allows more custom definition of refinement regions
+    from clawpack.amrclaw.data import FlagRegion
+    flagregion = FlagRegion(num_dim=2)
+    flagregion.name = 'Region_LatexShelf'
+    flagregion.minlevel = 2
+    flagregion.maxlevel = 5
+    flagregion.t1 = 0.
+    flagregion.t2 = 1e9
+    flagregion.spatial_region_type = 2 # Ruled Rectangle
+    
 
-    #USS Lexington
-    rundata.gaugedata.gauges.append([5, -97.38, 27.80,
-				     rundata.clawdata.t0,
-                                     rundata.clawdata.tfinal])
+    from numpy import ma # masked arrays
+    from clawpack.visclaw import colormaps, plottools
+    from clawpack.geoclaw import topotools, marching_front
+    from clawpack.amrclaw import region_tools
+    import clawpack.geoclaw.topotools as topo
 
-    # Force the gauges to also record the wind and pressure fields
-    rundata.gaugedata.aux_out_fields = [4, 5, 6]
+    zmin = -60.
+    zmax = 40.
 
+    land_cmap = colormaps.make_colormap({ 0.0:[0.1,0.4,0.0],
+                                        0.25:[0.0,1.0,0.0],
+                                        0.5:[0.8,1.0,0.5],
+                                        1.0:[0.8,0.5,0.2]})
+
+    sea_cmap = colormaps.make_colormap({ 0.0:[0,0,1], 1.:[.8,.8,1]})
+
+    cmap, norm = colormaps.add_colormaps((land_cmap, sea_cmap),
+                                        data_limits=(zmin,zmax),
+                                        data_break=0.)
+                                        
+    sea_cmap_dry = colormaps.make_colormap({ 0.0:[1.0,0.7,0.7], 1.:[1.0,0.7,0.7]})
+    cmap_dry, norm_dry = colormaps.add_colormaps((land_cmap, sea_cmap_dry),
+                                        data_limits=(zmin,zmax),
+                                        data_break=0.)
+
+    topo_file = topo.Topography()
+    topo_file.read('../../../scratch/gulf_caribbean.tt3', topo_type=3)
```

Comment:
May be better to use the `CLAW` environment variable and refer to this file that way.  Note for instance this will fail if not being run from the examples directory in GeoClaw.